### PR TITLE
RHDEVDOCS-4475: understanding odo

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-4475

Link to docs preview:
https://51136--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/developer_cli_odo/understanding-odo.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: cli_reference/developer_cli_odo/understanding-odo.adoc [hunk 1]
```diff
@@ -6,11 +6,30 @@ include::_attributes/common-attributes.adoc[]
...
+
+The inner loop primarily takes place on your development computer whereas the outer loop typically runs on shared servers, and is often automated with continuous integration and continuous deployment (CI/CD) pipelines.
+
+== Benefits of odo
+
+If you are an application developer wishing to deploy to Kubernetes, `odo` will help if:
+
+- You rely on frameworks such as Node.js, Java Spring Boot, and .NET.
+- Your application will run on Kubernetes-like infrastructure.
+- You do not want to learn Kubernetes and DevOps, to deploy to enterprise infrastructure.
```
Comment (Preeticp) [inline cli_reference/developer_cli_odo/understanding-odo.adoc:27]: How about: You want to deploy to enterprise infrastructure though you might not have a deep knowledge of Kubernetes and DevOps.

**Review by Preeticp — COMMENTED** [review]
Looks great overall. Left a few suggestions, PTAL.

File: cli_reference/developer_cli_odo/understanding-odo.adoc [hunk 2]
```diff
@@ -6,11 +6,30 @@ include::_attributes/common-attributes.adoc[]
...
+
+As an application developer wanting to deploy to Kubernetes, `odo` helps if:
+
+- You rely on frameworks such as Node.js, Java Spring Boot, and .NET.
+- You want to run your application on Kubernetes-like infrastructure.
+- You want to deploy to enterprise infrastructure even though you might not have a deep knowledge of Kubernetes and DevOps.
+
+== Comparison with kubectl and oc
+
+`odo` focuses on application developers and cloud engineers, whereas `kubectl` and `oc`, the command line interface for {product-title}, are DevOps tools. The latter help you to deploy applications to a cluster, and to support on-going maintenance, provided you already have a good knowledge of Kubernetes. `odo` does not support system administration or maintenance tasks for production clusters.
```
Comment (valaparthvi) [inline cli_reference/developer_cli_odo/understanding-odo.adoc:31]:
```suggestion
`odo` focuses on application developers and cloud engineers, whereas `kubectl` and `oc`, the command line interface for {product-title}, are DevOps tools. The latter help you to deploy applications to a cluster, and to support on-going maintenance, provided you already have a good knowledge of Kubernetes. `odo` does not support system administration or maintenance tasks for production clusters, but it does not require you to have a deep understanding of Kubernetes if you want to develop in the cluster.
```

**Review by valaparthvi — CHANGES_REQUESTED** [review]

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by valaparthvi** [comment]
@gabriel-rh since we're going to un-publish odo docs from the upstream docs, do you think it might be time to close this PR and the others?

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request closed without merging**